### ISSUE TITLE
Multiple districts search

### ISSUE THREAD

**Issue by anant0206** [sushrut111/cowin-automation-extn#98]
Can there be an option to search for multiple districts within a state? For example, in Delhi - one district has only 2-3 relevant centers, but all districts are equally accessible.

**Comment by Shweta200126** [comment]
@anant0206 just to add, I think it's a very nice idea but since cowin website only allows one district at a time, maybe we can do a kind of loop refreshing for districts. Like it refreshes page after 15 sec suppose and changes it to another district, so one would know if there is availabilty in one district or another or not.

**Comment by AshishNamdev** [comment]
> @anant0206 just to add, I think it's a very nice idea but since cowin website only allows one district at a time, maybe we can do a kind of loop refreshing for districts. Like it refreshes page after 15 sec suppose and changes it to another district, so one would know if there is availabilty in one district or another or not.

In such case, if vaccine is not available in one district but available in another, it will be selected and proceed for booking the slot.
If user have selected `Auto Confirm` in the form ( which is the case for most of the users) the slot will be booked in another district, which user might not want, and vaccine slot will be wasted.

So it will not serve the purpose.

**Comment by Shweta200126** [comment]
> > @anant0206 just to add, I think it's a very nice idea but since cowin website only allows one district at a time, maybe we can do a kind of loop refreshing for districts. Like it refreshes page after 15 sec suppose and changes it to another district, so one would know if there is availabilty in one district or another or not.
> 
> In such case, if vaccine is not available in one district but available in another, it will be selected and proceed for booking the slot.
> If user have selected `Auto Confirm` in the form ( which is the case for most of the users) the slot will be booked in another district, which user might not want, and vaccine slot will be wasted.
> 
> So it will not serve the purpose.

No, look at it this way; where I live I can easily travel between two districts but I don't have the option for it which is why I said it will be beneficial for people like me.

**Comment by AshishNamdev** [comment]
> > > @anant0206 just to add, I think it's a very nice idea but since cowin website only allows one district at a time, maybe we can do a kind of loop refreshing for districts. Like it refreshes page after 15 sec suppose and changes it to another district, so one would know if there is availabilty in one district or another or not.
> > 
> > 
> > In such case, if vaccine is not available in one district but available in another, it will be selected and proceed for booking the slot.
> > If user have selected `Auto Confirm` in the form ( which is the case for most of the users) the slot will be booked in another district, which user might not want, and vaccine slot will be wasted.
> > So it will not serve the purpose.
> 
> No, look at it this way; where I live I can easily travel between two districts but I don't have the option for it which is why I said it will be beneficial for people like me.

In conditions like this, there will be very few limited number of users for this feature.

Also keep in mind Cowin portal allows limited number of search after which it forcefully log out the user and disable `Get OTP` button for 180 seconds.

I suggest we should close this issue and open Discussion about this.

**Comment by sushrut111** [comment]
I agree with Ashish, there isn't a lot of utility for the feature.
But if one wants to work on it, I am okay with that.
